### PR TITLE
adding specs for Account API

### PR DESCRIPTION
**Pull request by Tokesh**
Specs for [Account API](https://opensearch.org/docs/2.7/security/access-control/api/#account)
### Issues Resolved
[#106]

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: model/common_structures.smithy [hunk 1]
```diff
@@ -59,3 +59,8 @@ structure Hits{
     _source: Document,
     fields: Document
 }
+
+structure Response{
+    status: String
+    message: String
+}
```
Comment (nhtruong) [inline model/common_structures.smithy:66]: A structure named response is very vague. Any Response body can be Response

Comment (Tokesh) [reply model/common_structures.smithy:66]: > A structure named response is very vague. Any Response body can be Response

Can we name it "UpdateResponse"?

Comment (Tokesh) [reply model/common_structures.smithy:66]: because we use it in many API's in update/delete parts

**Review by nhtruong — APPROVED** [review]
🚀

**Review by VachaShah — APPROVED** [review]

**Pull request merged**